### PR TITLE
Fix minor bugs found with coverity

### PR DESCRIPTION
**Pull request by abolinsky**
# MERGE https://github.com/rapyuta-robotics/alica/pull/15 FIRST

### CODE REVIEW

File: alica_engine/src/engine/Logger.cpp [hunk 1]
```diff
@@ -23,7 +23,7 @@ Logger::Logger(AlicaEngine* ae) {
     supplementary::SystemConfig* sc = supplementary::SystemConfig::getInstance();
     this->active = (*sc)["Alica"]->get<bool>("Alica.EventLogging.Enabled", NULL);
     if (this->active) {
-        char buffer[50];
+        char buffer[1024];
```
Comment (hendrik-skubch) [inline alica_engine/src/engine/Logger.cpp]: No, the buffer does not need to be this large, strftime produces a result with a fixed length.
What's more, neither buffer nor timeString are actually used, they can be removed entirely.

File: alica_engine/src/engine/AlicaEngine.cpp [hunk 2]
```diff
@@ -23,38 +23,41 @@
...
-        , terminating(false)
-        , expressionHandler(nullptr)
-        , log(nullptr)
-        , auth(nullptr)
-        , variableSyncModule(nullptr)
-        , pap(nullptr)
-        , stepEngine(stepEngine)
-        , agentIDManager(idManager) {
+AlicaEngine::AlicaEngine(supplementary::AgentIDManager* idManager, const std::string& roleSetName, const std::string& masterPlanName,
+                         const std::string& roleSetDir, bool stepEngine)
```
Comment (abolinsky) [inline alica_engine/src/engine/AlicaEngine.cpp]: Some crazy formatting issue

Comment (gauthamm) [reply alica_engine/src/engine/AlicaEngine.cpp]: Can you manually correct it if the formatter does not work properly.

**Comment by hendrik-skubch** [comment]
The formatter works as intended, what is the issue?

**Pull request merged**